modules/nw-egress-ips-assign.adoc

```diff
@@ -49,3 +49,10 @@ egressips.k8s.ovn.org/<egressips_name> created
 ----
 
 . Optional: Save the `<egressips_name>.yaml` file so that you can make changes later.
+. Add labels to the namespace that requires egress IP addresses. To add a label to the namespace of an `EgressIPs` object defined in step 1, run the following command:
```

Comment:
```suggestion
. Add labels to the namespace that requires egress IP addresses. To add a label to the namespace of an `EgressIP` object defined in step 1, run the following command:
```

Reply:
@ctauchen I have chnaged `EgressIPs` to `EgressIP`